frontend/src/pages/Profile.js

```diff
@@ -1,17 +1,57 @@
 import PageComponent from '@component/PageComponent.js';
+import UserProfile from '@component/contents/UserProfile.js';
+import openModalButton from '@component/button/OpenModalButton';
+import modalComponent from '@component/modal/ModalComponent';
+import editProfile from '@component/contents/EditProfile';
 
 class Profile extends PageComponent {
   constructor() {
     super();
     this.setTitle('Profile');
   }
 
+  async getUserInfo() {
+    const userProfileInfo = await fetch('http://localhost:8080/userInfo').then(
+      (res) => res.json()
+    );
+    return userProfileInfo;
+  }
+
   async render() {
+    const editModalBtn = openModalButton({
+      text: 'EDIT PROFILE',
```

Comment:
```suggestion
      text: '>> EDIT PROFILE <<',
```
화살표 넣어주면 버튼 느낌이 더 날 것 같아요!